client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -179,6 +181,12 @@ class SiteSettingsFormDiscussion extends Component {
 					onChange={ this.handleCommentOrder }>
 					<span>{ translate( 'Comments should be displayed with the older comments at the top of each page' ) }</span>
 				</CompactFormToggle>
+				<JetpackModuleToggle
+					disabled={ isRequestingSettings || isSavingSettings }
+					label={ translate( 'Enable pop-up business cards over commenters’ Gravatars' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **9** times:
`translate( 'Enable pop-up business cards over commenters’ Gravatars.' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).